kbase-extension/static/kbase/js/util/util.js

```diff
@@ -60,9 +60,15 @@ define(['bluebird'], (Promise) => {
         }
     }
 
+    function objectToString(obj) {
+        const type = Object.prototype.toString.call(obj);
+        return type.substring(8, type.length - 1);
+    }
```

Comment:
amazing new function to determine what an object is

Reply:
I like the simplicity, but it's maybe nice to go with some constants here, too. Like, using it as:
`if (Util.objectToString(thing) === Util.OBJECT) { do stuff }`

...and now that I type it, that's probably overkill, and will be redundant once we move to TypeScript.